### PR TITLE
fix ownership of config files; change homepage; change spec point to …

### PR DESCRIPTION
**Pull request by zvada**
* fix ownership of config files in spec
* change in spec point to proper redirector hostname
* change homepage in Origin config
* fix current version in `/usr/sbin/stashcache`

### CODE REVIEW

File: spec/stashcache.spec [hunk 1]
```diff
@@ -53,26 +49,31 @@ Group: Grid
...
 mkdir -p %{buildroot}%{_sysconfdir}/xrootd
 make install DESTDIR=%{buildroot}
-for src in "./configs/xrootd-stashcache-origin-server.cfg.in" \ 
-           "./configs/xrootd-stashcache-cache-server.cfg.in"; do
+for src in "./configs/xrootd-stashcache-origin-server.cfg.in" "./configs/xrootd-stashcache-cache-server.cfg.in"; do
     dst=$(basename "$src" .cfg.in)
     sed -i -e "s#@LIBDIR@#%{_libdir}#" "$src"
     sed -e "s#@REDIRECTOR@#%{redirector_prod}#" \
         "$src" > "%{buildroot}%{_sysconfdir}/xrootd/${dst}.cfg"
+    chown xrootd:xrootd %{buildroot}%{_sysconfdir}/xrootd/${dst}.cfg
```
Comment (bbockelm) [inline spec/stashcache.spec]: This isn't the correct way to set the UID as it hardcodes the UID from the build machine.

Instead, you want to assign ownership in the `%files` section.

Comment (zvada) [inline spec/stashcache.spec]: Instead, would this do a job simply add line per origin-server and cache-server `%files` section like:
`%defattr(-,xrootd,xrootd)`

Comment (bbockelm) [inline spec/stashcache.spec]: yup, that's the idea!

Comment (zvada) [inline spec/stashcache.spec]: Thanks, did proper settings in this commit: https://github.com/opensciencegrid/StashCache-Daemon/pull/4/commits/977c99c939b1918f0aa3109db9cf5b560a1e8095

Comment (bbockelm) [inline spec/stashcache.spec]: Wait, now I'm more confused.

Why is the config file owned by the daemon user?  Doesn't that mean the daemon can change its own configs?

Comment (zvada) [inline spec/stashcache.spec]: It turns out xrootd sets ownership of default config files to `xrootd:xrootd` which seems incorrect. Previuosly it couldn't read configs NOT owned by `xrootd:xrootd` but test in 4.6.1 it's not a case anymore. I've reverted and leave root:root set for StashCache config files and will ignore what xrootd ships by default.

Comment (zvada) [inline spec/stashcache.spec]: @bbockelm: are you ok with this and can I squash and merge?

**Pull request merged**